pkg/reconciler/pipelinerun/resources/apply.go

```diff
@@ -318,9 +318,14 @@ func replaceParamValues(params []v1beta1.Param, stringReplacements map[string]st
 func ApplyTaskResultsToPipelineResults(
 	results []v1beta1.PipelineResult,
 	taskRunResults map[string][]v1beta1.TaskRunResult,
-	customTaskResults map[string][]v1beta1.CustomRunResult) ([]v1beta1.PipelineRunResult, error) {
+	customTaskResults map[string][]v1beta1.CustomRunResult, skipTasks []v1beta1.SkippedTask) ([]v1beta1.PipelineRunResult, error) {
```

Comment:
My comment might be wrong, but is this code correctly formatted like this?

Reply:
+1

Reply:
Oh thanks!